src/realm/realmReducer.js

```diff
@@ -76,6 +76,9 @@ export default (state: RealmState = initialState, action: Action): RealmState =>
     }
 
     // TODO on EVENT_USER_UPDATE for self: update email, isAdmin, etc.
+    // TODO on EVENT_USER_UPDATE for a user in
+    // `state.realm.nonActiveUsers`: Convert `.avatar_url` field to
+    // AvatarURL instance.
```

Comment:
I think the real TODO task here is to update the user at all, which we don't.

When we do take on that task... hmm, in fact won't this conversion already be taken care of by the logic this branch adds in eventToAction?

Also this probably simply can't happen -- it's not clear there would ever be a reason for a user update event on a deactivated user, which is what these are.

Reply:
Ah, right! Thanks!